tools/pd-ctl/pdctl/command/region_command.go

```diff
@@ -303,8 +324,7 @@ func showRegionsFromStartKeyCommandFunc(cmd *cobra.Command, args []string) {
 		fmt.Println("Error: unknown format")
 		return
 	}
-	// TODO: Deal with path escaped
-	prefix := regionKeyPrefix + "/" + key
+	prefix := regionKeyPrefix + "/" + url.QueryEscape(key)
```

Comment:
Do we need to update server-side to adapt this change?

Reply:
It seems #1299 will deal with this problem. I will remove it from this PR.